packages/react-core/src/components/Slider/examples/Slider.md

```diff
@@ -133,14 +134,23 @@ class ContinuousInput extends React.Component {
   render() {
     return (
       <>
-        <Text component={TextVariants.h3}>Slider value is: {this.state.value.toFixed(2)}</Text>
-        <Slider value={this.state.value} onChange={this.onChange} />
+        <Checkbox
+          id="thumb-has-tooltip"
```

Comment:
I would add a style={{ marginBottom: 20 }} to the Checkbox.

Reply:
Added.